src/Illuminate/Foundation/Exceptions/views/5xx.blade.php

```diff
@@ -0,0 +1,11 @@
+@extends('errors::minimal')
+
+@php
+    $statusCode = $exception->getStatusCode();
+    $statusText = __(\Symfony\Component\HttpFoundation\Response::$statusTexts[$statusCode]);
+    $statusMessage = !empty($exception->getMessage()) ? ': ' . __($exception->getMessage()) : '';
```

Comment:
For a 5xx, the user has absolutely know business seeing the exception message, that's for sure.